### ISSUE TITLE
Parse errors on a beatmap

### ISSUE THREAD

**Issue by nilesr** [fmang/oshu#45]
First time trying oshu out, maybe it's been broken for a while, but I parse warnings (in attached log) when trying to play the attached beatmap. Only about 1/10th of the notes actually appear in the window.
[all.log](https://github.com/fmang/oshu/files/1633239/all.log)
[3L - Three Magic (cRyo[iceeicee]) [Kite's Hard].osu.txt](https://github.com/fmang/oshu/files/1633242/3L.-.Three.Magic.cRyo.iceeicee.Kite.s.Hard.osu.txt)

**Comment by fmang** [comment]
I've played oshu! on many beatmaps from the recent beatmap packs. However, the beatmap you tried is pretty old, and contains things the newer editors would never generate.

Since there's no complete reference on the old formats, the best way to support them is to try them out like you did. Thank you for reporting by the way :)

I have corrected the beatmap parser to support that beatmap, and played your beatmap to ensure it worked. Pull the master branch to get the fix. I'll release it with 1.6.0.
